backend/app/Models/Transaction.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
The price we are storing in the listings table is converted to a different unit, you will want to add a getter method that does the conversion. That way it comes out in the right format

```
public function getPriceAttribute($value)
    {
        //price is stores as a bigint in the db, need to make it a properly formatted float.
        return EthereumConverter::convertETHPriceToFloat($value);
    }
```
[Nicky made one here](https://github.com/CryptoCardsETH/cryptocards/blob/25edba4d2ec282cbd981fb419809056f1e340f62/backend/app/Models/Listing.php#L18-L22)

Reply:
You might want to make a setter too and convert it to an Int so that when you take the price out of listings you don't end up storing a float in transactions
[convertETHPriceToInt($ethPriceFloat)](https://github.com/CryptoCardsETH/cryptocards/blob/25edba4d2ec282cbd981fb419809056f1e340f62/backend/app/Helpers/EthereumConverter.php)

Reply:
The price should remain a bitInteger everywhere on the backend - no setter should be needed.

But when the price of the transaction is returned to the frontend, it should be returned as a float.

Reply:
*bigInteger but yeah